src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -1432,6 +1436,42 @@ QString MainWindow::get_filter_string(const int filter)
     return filters.join(";;");
 }
 
+void MainWindow::initialize_ocio()
+{
+    // Try first a user specified OCIO config.
+    if (getenv("OCIO"))
+    {
+        try
+        {
+            m_ocio_config = OCIO::GetCurrentConfig();
+            RENDERER_LOG_INFO("Using OpenColorIO config: %s", getenv("OCIO"));
```

Comment:
Our (weird, admittedly) convention is that log messages are all lower-case. Also we use `oiio` instead of `OpenImageIO` everywhere so let's do the same for now:

    RENDERER_LOG_INFO("using ocio configuration %s", getenv("OCIO"));

Maybe one day I'll fix the case everywhere... Until then, let's try to be consistent.

Same change below.